test/testvm.py

```diff
@@ -752,45 +750,41 @@ def _cleanup(self):
             print >> sys.stderr, "WARNING: Cleanup failed:", str(value)
 
     def kill(self):
+        # attempt graceful shutdown first
```

Comment:
I think we shouldn't do this.  A machine is either shut down cleanly via the `shutdown` method or killed unceremoniously.  It is shutdown when it is in 'maintenance' mode and we want to keep the changes to its file system after we are done with it, such as with vm-install.  After a normal test that doesn't change the image at all, we can just kill qemu.  That ought to be faster and more reliable.